cluster/manifests/z-kubenurse/daemonset.yaml

```diff
@@ -0,0 +1,83 @@
+{{if eq .Cluster.ConfigItems.network_monitoring_enabled "true" }}
+---
+# Source: kubenurse/templates/daemonset.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    application: kubernetes
+    component: kubenurse
+  name: kubenurse
+  namespace: kubenurse
+spec:
+  selector:
+    matchLabels:
+      application: kubernetes
+      component: kubenurse
+  template:
+    metadata:
+      labels:
+        application: kubernetes
+        component: kubenurse
+      annotations:
+        prometheus.io/path: "/metrics"
+        prometheus.io/port: "8080"
+        prometheus.io/scheme: "http"
+        prometheus.io/scrape: "true"
+    spec:
+      serviceAccountName: "kubenurse"
+      containers:
+        - name: "kubenurse"
+          env:
+            - name: KUBENURSE_INGRESS_URL
+              value: https://kubenurse.{{ .Values.hosted_zone }}
+            - name: KUBENURSE_SERVICE_URL
+              value: "http://kubenurse.kubenurse.svc.cluster.local:8080"
+            - name: KUBENURSE_INSECURE
+              value: "true"
+            - name: KUBENURSE_NAMESPACE
+              value: kubenurse
+            - name: KUBENURSE_NEIGHBOUR_FILTER
+              value: "application=kubernetes,component=kubenurse"
+            - name: KUBENURSE_ALLOW_UNSCHEDULABLE
+              value: "{{.Cluster.ConfigItems.network_monitoring_check_unschedulable_nodes}}"
+            - name: KUBENURSE_CHECK_INTERVAL
+              value: "{{.Cluster.ConfigItems.network_monitoring_check_interval}}"
+            - name: KUBENURSE_CHECK_API_SERVER_DIRECT
+              value: "{{.Cluster.ConfigItems.network_monitoring_check_api_server_direct}}"
+            - name: KUBENURSE_CHECK_API_SERVER_DNS
+              value: "{{.Cluster.ConfigItems.network_monitoring_check_api_server_dns}}"
+            - name: KUBENURSE_CHECK_ME_SERVICE
+              value: "{{.Cluster.ConfigItems.network_monitoring_check_kubenurse_service}}"
+            - name: KUBENURSE_CHECK_ME_INGRESS
+              value: "{{.Cluster.ConfigItems.network_monitoring_check_kubenurse_ingress}}"
+            - name: KUBENURSE_CHECK_NEIGHBOURHOOD
+              value: "{{.Cluster.ConfigItems.network_monitoring_check_neighborhood}}"
+          image: "v1.6.1-0.20221026195314-d63aadd264e8-main-1"
+          resources:
+            requests:
+              cpu: "40m"
+              memory: "100Mi"
+            limits:
+              cpu: "40m"
+              memory: "100Mi"
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+          readinessProbe:
+            failureThreshold: 1
+            httpGet:
+              path: /ready
+              port: 8080
+              scheme: HTTP
+            periodSeconds: 3
+            successThreshold: 1
+            timeoutSeconds: 1
+      tolerations:
+        - effect: NoSchedule
+          key: node.kubernetes.io/role
+          value: master
+        - operator: Exists
+          effect: NoSchedule
```

Comment:
This makes the other toleration redundant I believe. We can also use the pattern that we use for all other DaemonSets that should run everywhere, e.g. [flannel](https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/flannel/daemonset.yaml#L127-L131):

```yaml
      tolerations:
      - operator: Exists
        effect: NoSchedule
      - operator: Exists
        effect: NoExecute
```